views/js/util/forms/inputBehaviours.js

```diff
@@ -0,0 +1,181 @@
+/**
```

Comment:
I'm not super fan of such general component name, since it does not say anything about the "behavior".  And it does not manage strictly only input behaviours. I'm thinking about something like deliveriesFormBehaviour or more concrete. What do you think?

Reply:
Changing to `forms/deliveryFormHelper`.